dataplane/handlers/interface.go

```diff
@@ -247,7 +251,16 @@ func (ni *Interface) reconcile(config *oc.Interface) {
 	if config.GetOrCreateSubinterface(0).Enabled != nil {
 		if state.GetOrCreateSubinterface(0).Enabled == nil || config.GetSubinterface(0).GetEnabled() != state.GetSubinterface(0).GetEnabled() {
 			log.V(1).Infof("setting interface %s enabled %t", tapName, config.GetSubinterface(0).GetEnabled())
-			if err := ni.ifaceMgr.SetState(tapName, config.GetSubinterface(0).GetEnabled()); err != nil {
+			state := fwdpb.PortState_PORT_STATE_DISABLED_DOWN
+			if config.GetSubinterface(0).GetEnabled() {
+				state = fwdpb.PortState_PORT_STATE_ENABLED_UP
+			}
+			_, err := ni.e.PortState(context.Background(), &fwdpb.PortStateRequest{
```

Comment:
Why is the return discarded?

Reply:
because the state update happens when a  notification is received, don't need to do twice